app/javascript/components/pages/LastCallNew.js

```diff
@@ -1,154 +1,166 @@
-import React from 'react'
-import { useState, useRef } from 'react'
-import { Form, Button, Container, Col, Row } from 'react-bootstrap'
-import { Navigate } from 'react-router-dom'
+import React from "react";
+import { useState, useRef } from "react";
+import { Form, Button, Container, Col, Row } from "react-bootstrap";
+import { Navigate } from "react-router-dom";
 
 export default function LastCallNew(props) {
+  let { createCocktail, current_user, logged_in } = props;
```

Comment:
restructuring can also be done on the line above from props.

Reply:
How would i get current_user, it's being passed as props from rails devise?

Reply:
ment destructuring 

```export default function LastCallNew({createCocktail, current_user, logged_in}) {```

^ is the same thing as line 6